dev/release/create-tarball.sh

```diff
@@ -53,8 +53,7 @@ fi
 
 object_store_version=$1
 rc=$2
-
-tag=object_store_${object_store_version}
+tag=v${object_store_version}
```

Comment:
I haven't tested this -- I expect I will do so as part of actually creating the RC